backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -371,7 +372,21 @@ def fetch_contents_ide(self, request: HttpRequest, pk: Any = None) -> Response:
         if not file_path.endswith("/"):
             file_path += "/"
         file_path += file_name
-        contents = FileManagerHelper.fetch_file_contents(file_system, file_path)
+        # Temporary Hack for frictionless onboarding as the user id will be empty
+        try:
+            contents = FileManagerHelper.fetch_file_contents(file_system, file_path)
+        except FileNotFound:
+            file_path = file_path = FileManagerHelper.handle_sub_directory_for_tenants(
+                UserSessionUtils.get_organization_id(request),
+                is_create=True,
+                user_id="",
+                tool_id=str(custom_tool.tool_id),
+            )
+            if not file_path.endswith("/"):
+                file_path += "/"
+                file_path += file_name
```

Comment:
NIT: Good to use pathlib

Reply:
Used existing way of handling. Will handle it in different PR.